src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -165,8 +165,8 @@ class PopoverReportActionContextMenu extends React.Component {
                     vertical: nativeEvent.pageY - y,
                 },
                 popoverAnchorPosition: {
-                    horizontal: nativeEvent.pageX,
-                    vertical: nativeEvent.pageY,
+                    horizontal: nativeEvent.pageX + CONST.MODAL.POPOVER_ANCHOR_SHIFT,
+                    vertical: nativeEvent.pageY - CONST.MODAL.POPOVER_ANCHOR_SHIFT,
```

Comment:
is this a solution for https://expensify.slack.com/archives/C049HHMV9SM/p1691745427354299? if so this is not fixing the root cause, the hover effect will be lost when you move the mouse out, but it will stay initially right? please correct me if i am wrong.

Reply:
yes you're right, that's what i thought initially, the fact that the hover wasn't lost the first time made me suspect that the issue isn't caused by that and it wasn't. additionally the hover should not be lost if the popover is active and mouseLeave triggers.